src/drivers/sdl/lv_sdl_window.c

```diff
@@ -86,7 +86,7 @@ lv_display_t * lv_sdl_window_create(int32_t hor_res, int32_t ver_res)
         SDL_Init(SDL_INIT_VIDEO);
         SDL_StartTextInput();
         event_handler_timer = lv_timer_create(sdl_event_handler, 5, NULL);
-        lv_tick_set_cb(SDL_GetTicks);
+        //        lv_tick_set_cb(SDL_GetTicks);
```

Comment:
Nit: Functions that are not needed can be safely removed, if there is  plan to reenable in very near future, please put a comment here explaining why it was removed for now.

Reply:
Upps! It was commented out when I was debugging something. Thanks! :100: